src/common/components/failedVerificationModal/failedVerificationModal.component.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular'
+import template from './failedVerificationModal.tpl.html'
+
+const componentName = 'failedVerificationModal'
+
+class FailedVerificationModalController {
+  /* @ngInject */
+  constructor () /* eslint-disable-line no-useless-constructor */ {}
+}
+
+export default angular
+  .module(componentName, [])
+  .component(componentName, {
+    controller: FailedVerificationModalController,
```

Comment:
I think you can just delete the controller if it's not needed.
```suggestion

```

Reply:
I'll give it a try. Wasn't sure if directives had to have controllers backing them. Was basing this off of the retry modal.

Reply:
We're using `$ctrl.title` and `$ctrl.onOk()`. How would I do those calls if I don't have a controller?

Reply:
Does it break? I assume it attaches the bindings to the view model. I don't think a controller is necessary but I could be wrong. The `this` of the controller is the same as the view model. AngularJS is confusing...